docs-src/python/how-to-use-worker-versioning-in-python.md

```diff
@@ -0,0 +1,101 @@
+---
+id: how-to-use-worker-versioning-in-python
+title: How to use Worker Versioning in Python
+sidebar_label: Worker Versioning
+description: Version your Python Workers by using build ID-based versioning
+tags:
+  - python
+  - how-to
+---
+
+To use [Worker Versioning](/workers#worker-versioning) in Python, you need to do the following:
```

Comment:
May need a bit more obvious link to the concepts section as it can almost be a prerequisite before getting into Python specifics